bin/alias.bat

```diff
@@ -1,27 +1,31 @@
 @echo off
 
 set ALIASES=%CMDER_ROOT%\config\aliases
+setlocal
+:: handle quotes within command definition, e.g. quoted long file names
+set _x="%*"
+set _x=%_x:"=%
 
-if ["%*"] == [""] echo Use /? for help & echo. & goto :p_show
+:: check command usage
+if ["%_x%"] == [""] echo Use /? for help & echo. & goto :p_show
 if ["%1"] == ["/?"] goto:p_help
 if ["%1"] == ["/reload"] goto:p_reload
 :: /d flag for delete existing alias
 if ["%1"] == ["/d"] goto:p_del %*
 if ["%2"] == [""] echo Insufficient parameters. & goto:p_help
-::validate alias
-setlocal
-for /f "delims== tokens=1" %%G in ("%*") do set _temp2=%%G
 
-	set _temp=%_temp2: =%
+:: validate alias
+for /f "delims== tokens=1" %%G in ("%_x%") do set alias=%%G
```

Comment:
IMHO, since `%_x%` usage is not needed in arguments check (line 10) then it is better to move its definition to actual usage here.